### PR TITLE
Add getDefaultNode

### LINKED ISSUE #8: Implement .getDefaultNode()
Implement the api endpoint getDefaultNode which returns the mesh node from where the user is connected to the dashboard if there is any or null otherwise. The node should be a string, like 'ql-roxa'

### PR DESCRIPTION
**Pull request by gmarcos87**
Api to get from which node I am connected, in case of error the answer is null

Close #8

### CODE REVIEW

File: web-app/src/api/restApi.js [hunk 1]
```diff
@@ -143,6 +143,11 @@ const api = {
             }, {...config, path: '/reports/create'})
         },
     },
+    nodes: {
+        getDefaultNode: () => fetch('http://thisnode.info/cgi-bin/hostname')
```
Comment (germanferrero) [inline web-app/src/api/restApi.js]: In my current network this fetch doesn't fail (and triggers catch), but hangs for ever. I think we can add a timeout here so when the user isn't connected to the mesh we can show a friendly message.

Comment (gmarcos87) [reply web-app/src/api/restApi.js]: Does fetch have a timeout?

Comment (germanferrero) [reply web-app/src/api/restApi.js]: no, I think we should add our own timeout

Comment (gmarcos87) [reply web-app/src/api/restApi.js]: You can try now?

**Pull request merged**